scalene/scalene_preload.py

```diff
@@ -31,9 +54,6 @@ def setup_preload(args: argparse.Namespace) -> bool:
         # Load shared objects (that is, interpose on malloc, memcpy and friends)
         # unless the user specifies "--cpu-only" at the command-line.
 
-        if args.cpu_only:
```

Comment:
Right now, we need this for Jupyter notebooks since preloading doesn't yet work there.

Reply:
That check is still there, in `get_preload_environ()`.  We don't set up the preloading environment variables if `cpu_only`, and if no environment variables need overriding, we don't start a new process.

On MacOS, we need python to be started with `OBJC_DISABLE_INITIALIZE_FORK_SAFETY` defined to support multiprocessing.

Reply:
BTW, I tested with Jupyter on my mac, and the memory profiling seemed to work...

Reply:
Wow, that'd be awesome.